react/LocationSelect.tsx

```diff
@@ -112,7 +115,10 @@ const LocationSelect: React.FC = () => {
         </div>
       )
 
-      currentOptions = fieldValue ? currentOptions[fieldValue as keyof {}] : {}
+      currentOptions = fieldValue
+        ? // eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
I hate this rule 😆

Reply:
tbh I don't really get the difference of `object` vs `Record<string, unknown>` (not that I actually really searched for it tho), but what scared me a little bit is that `currentOptions` can also be an array, as shown on line 90